src/controllers/payment.controller.ts

```diff
@@ -34,8 +41,12 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
       logger.debugRequest(req, `Overseas Entity id: ${ appData[OverseasEntityKey] }, Payment status: ${status}, Redirecting to: ${CHECK_YOUR_ANSWERS_URL}`);
```

Comment:
This log message needs updating as it now doesn't always redirect to the 'check your answers' screen.